packages/eslint-plugin/src/rules/no-single-declare-module.ts

```diff
@@ -0,0 +1,58 @@
+import { ESLintUtils } from "@typescript-eslint/utils";
+import * as ts from "typescript";
+
+import { createRule } from "../util";
+
+const rule = createRule({
```

Comment:
Obligatory reminder to add this to the preset (tell me if you already know and I'll stop mentioning it in the future 😄)

Reply:
No, every single reminder is helpful.